parcels/particle.py

```diff
@@ -44,6 +46,8 @@ def __repr__(self):
 
     def is64bit(self):
         """Check whether variable is 64-bit"""
+        if self.name == 'CGridIndexSet':
+            return True
         return True if self.dtype == np.float64 or self.dtype == np.int64 else False
```

Comment:
Or the more pythonic `return True if self.dtype == np.float64 or self.dtype == np.int64 or self.name =='CGridIndexSet' else False`